src/main/java/apleasebot/tasks/Task.java

```diff
@@ -0,0 +1,32 @@
+package apleasebot.tasks;
+
+/**
+ * Abstract class to declare what methods Tasks should implement
+ * Also provides some methods for the children classes to inherit
+ */
+public abstract class Task {
+    protected String name;
```

Comment:
Perhaps name it "description" instead of "name" as it is more aligned with the intent behind it.
```suggestion
    protected String desc;
```